elife/config/etc-init-goaws-init.conf

```diff
@@ -0,0 +1,11 @@
+description "GOAWS will run a local AWS server that can be used by setting --endpoint=http://localhost:4100 to commands"
+respawn
+respawn limit 10 30
+kill timeout 70
+setuid {{ pillar.elife.deploy_user.username }}
+env HOME=/home/{{ pillar.elife.deploy_user.username }}
+instance $ID
```

Comment:
`instance` is used to launch multiple instance of the same process, but here we need only one, remove